packages/salesforcedx-vscode-apex-replay-debugger/src/adapter/debugConfigurationProvider.ts

```diff
@@ -38,7 +41,9 @@ export class DebugConfigurationProvider
     token?: vscode.CancellationToken
   ): vscode.ProviderResult<vscode.DebugConfiguration> {
     return this.asyncDebugConfig(config).catch(async err => {
-      return vscode.window.showErrorMessage(err.message).then(x => undefined);
+      return vscode.window
+        .showErrorMessage(err.message, { modal: true })
```

Comment:
This change allows us to show the error message at the top of vscode instead of at the bottom right corner.